extension/spark-doris-connector/src/main/java/org/apache/doris/spark/serialization/RowBatch.java

```diff
@@ -70,7 +70,8 @@ public void put(Object o) {
         }
     }
 
-    private int offsetInOneBatch = 0;
+    // offset for iterate the rowBatch
+    private int offsetInRowBatch = 0;
     private int rowCountInOneBatch = 0;
```

Comment:
rowCountInOneBatch could be a local variable？

Reply:
rowCountInOneBatch is also used in `addValueToRow` and `convertArrowToRowBatch`, so I think this is better for now

Reply:
rowCountInOneBatch is also used in `addValueToRow` and `convertArrowToRowBatch`, so I think this is better for now